File: test/fixtures/swift/get_with_browser_headers.swift
import Foundation

let url = URL(string: "http://localhost:28139/")!

var request = URLRequest(url: url)
request.setValue("gzip, deflate, sdch", forHTTPHeaderField: "Accept-Encoding")
request.setValue("en-US,en;q=0.8", forHTTPHeaderField: "Accept-Language")
request.setValue("Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36", forHTTPHeaderField: "User-Agent")
request.setValue("text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8", forHTTPHeaderField: "Accept")
request.setValue("http://www.wikipedia.org/", forHTTPHeaderField: "Referer")
request.setValue("GeoIP=US:Albuquerque:35.1241:-106.7675:v4; uls-previous-languages=%5B%22en%22%5D; mediaWiki.user.sessionId=VaHaeVW3m0ymvx9kacwshZIDkv8zgF9y; centralnotice_buckets_by_campaign=%7B%22C14_enUS_dsk_lw_FR%22%3A%7B%22val%22%3A%220%22%2C%22start%22%3A1412172000%2C%22end%22%3A1422576000%7D%2C%22C14_en5C_dec_dsk_FR%22%3A%7B%22val%22%3A3%2C%22start%22%3A1417514400%2C%22end%22%3A1425290400%7D%2C%22C14_en5C_bkup_dsk_FR%22%3A%7B%22val%22%3A1%2C%22start%22%3A1417428000%2C%22end%22%3A1425290400%7D%7D; centralnotice_bannercount_fr12=22; centralnotice_bannercount_fr12-wait=14", forHTTPHeaderField: "Cookie")
request.setValue("keep-alive", forHTTPHeaderField: "Connection")
let task = URLSession.shared.dataTask(with: request) { (data, response, error) in
    if let error = error {
        print("Error: \(error)")
    } else if let data = data {
        let str = String(data: data, encoding: .utf8)
        print("Received data:\n\(str ?? "")")
    }
}

task.resume()

File: test/fixtures/r-httr2/get_with_browser_headers.r
library(httr2)

request("http://localhost:28139/") |> 
  req_headers(
    `Accept-Encoding` = "gzip, deflate, sdch",
    `Accept-Language` = "en-US,en;q=0.8",
    `User-Agent` = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36",
    `Accept` = "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
    `Referer` = "http://www.wikipedia.org/",
    `Connection` = "keep-alive"
  ) |> 
  req_headers(
    `GeoIP` = "US:Albuquerque:35.1241:-106.7675:v4",
    `uls-previous-languages` = '["en"]',
    `mediaWiki.user.sessionId` = "VaHaeVW3m0ymvx9kacwshZIDkv8zgF9y",
    `centralnotice_buckets_by_campaign` = '{"C14_enUS_dsk_lw_FR":{"val":"0","start":1412172000,"end":1422576000},"C14_en5C_dec_dsk_FR":{"val":3,"start":1417514400,"end":1425290400},"C14_en5C_bkup_dsk_FR":{"val":1,"start":1417428000,"end":1425290400}}',
    `centralnotice_bannercount_fr12` = "22",
    `centralnotice_bannercount_fr12-wait` = "14"
  ) |> 
  req_perform()
